### PR TITLE
ESLint configured for CLI

### LINKED ISSUE #99: Configure eslint for CLI

### PR DESCRIPTION
**Pull request by AleF83**
Close #99

### CODE REVIEW

**Comment by omerlh** [comment]
Also, let's run ESlint in the CI?

**Comment by AleF83** [comment]
Yep, I thought about it also. I'll do this

**Comment by AleF83** [comment]
@omerlh, approve the PR please

File: ci/codefresh_cli.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 version: '1.0'
 steps:
+  Eslint:
```
Comment (omerlh) [inline ci/codefresh_cli.yml]: Why a new step and not after the unit tests?

Comment (AleF83) [reply ci/codefresh_cli.yml]: Single responsibility

Comment (omerlh) [reply ci/codefresh_cli.yml]: mmm ok, let's keep like that until we move to circle/github actions

**Pull request merged**